vivado/BUILD

```diff
@@ -1,3 +1,22 @@
+# Copyright 2023 Google LLC
```

Comment:
None of the vivado stuff was written by us. So maybe change to the bazel_rules_hdl authors or the like.

Reply:
Done!